tests/unit-js/src/test/scala/tests/js/JsSuite.scala

```diff
@@ -282,9 +282,9 @@ class JsSuite extends BaseMarkdownSuite {
       |println(jsdocs.ExampleJS.greeting)
       |```
     """.stripMargin,
-    """|error: no-dom.md:4 (mdoc generated code) object scalajs is not a member of package org
+    """|error: no-dom.md:4 (mdoc generated code) object dom is not a member of package org.scalajs
        |def run0(node: _root_.org.scalajs.dom.html.Element): Unit = {
-       |                          ^
+       |                                  ^
```

Comment:
I was puzzled why this changed, but I believe it's due to the introduction of these classes:
https://github.com/scala-js/scala-js/tree/main/javalibintf/src/main/java/org/scalajs/javalibintf

Reply:
Oh lovely, the error depends on the Scala version 😅